chart/iam-runtime-infratographer/values.yaml

```diff
@@ -78,6 +78,9 @@ restartPolicy: ""
 # ref: https://kubernetes.io/docs/concepts/storage/volumes/
 volumeMounts: {}
 
+# -- Mount path for the iam runtime socket
+socketVolumeMountPath: /var/iam-runtime/
```

Comment:
Should remove the trailing slash so the default config doesn't have an ill formatted socketPath with double slashes.

Reply:
(or trim / suffixes from this value in _configmap.tpl

Reply:
fixed (though, // is fine)